dockerfiles/vcpkg/build.sh

```diff
@@ -46,30 +49,43 @@ function doBuild() {
     # docker push hpccsystems/platform-build-$1:$GITHUB_BRANCH
 
     CMAKE_OPTIONS_EXTRA=""
-    if [ "$1" == "centos-7" ]; then
+    if [ "$1" == "centos-7*" ]; then
         CMAKE_OPTIONS_EXTRA="-DVCPKG_TARGET_TRIPLET=x64-centos-7-dynamic"
     elif [ "$1" == "amazonlinux" ]; then
         CMAKE_OPTIONS_EXTRA="-DVCPKG_TARGET_TRIPLET=x64-amazonlinux-dynamic"
     fi
     mkdir -p $HOME/.ccache
     docker run --rm \
         --mount source="$(pwd)",target=/hpcc-dev/HPCC-Platform,type=bind,consistency=cached \
+        --mount source="$(realpath ~)/.cache/vcpkg",target=/root/.cache/vcpkg,type=bind,consistency=cached \
```

Comment:
What is this change for.  Worth documenting related changes in the jira.

Reply:
Moves the cached vcpkg build location into the host machine, which in turn will speed up local calls to build.sh for any "vcpkg_overlay" contents.

Reply:
I added the change information into the commit message.